### PR TITLE
DOC: update the pandas.Series.str.strip docstring

### PR DESCRIPTION
**Pull request by techedlaksh**
Checklist for the pandas documentation sprint (ignore this if you are doing
an unrelated PR):

- [x] PR title is "DOC: update the <your-function-or-method> docstring"
- [x] The validation script passes: `scripts/validate_docstrings.py <your-function-or-method>`
- [x] The PEP8 style check passes: `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [x] The html version looks good: `python doc/make.py --single <your-function-or-method>`
- [x] It has been proofread on language by another sprint participant

Please include the output of the validation script below between the "```" ticks:

```
################################################################################
##################### Docstring (pandas.Series.str.strip)  #####################
################################################################################

Strip whitespaces from string in Series.

Strip whitespace (including newlines) or given string from
each string in the Series/Index from left and right sides. Equivalent to
:meth:`str.strip`.

Parameters
----------
to_strip : str or unicode
    String or unicode to strip in the given string.

Examples
--------
>>> # strip method
>>> s = pd.Series(['   This is a Test 1  '])
>>> s
0    This is a Test 1
dtype: object
>>> s = s.str.strip()
>>> s
0    This is a Test 1
dtype: object
>>> # lstrip method
>>> s1 = pd.Series(['This is another Test'])
>>> s1
0    This is another Test
dtype: object
>>> s1 = s1.str.lstrip('This')
>>> s1
0     is another Test
dtype: object
>>> # rstrip method
>>> s2 = pd.Series(['This is the last test'])
>>> s2
0    This is the last test
dtype: object
>>> s2 = s2.str.rstrip('test')
>>> s2
0     This is the last
dtype: object

Returns
-------
stripped : Series/Index of objects

################################################################################
################################## Validation ##################################
################################################################################

Errors found:
	See Also section not found


```

### CODE REVIEW

File: pandas/core/strings.py [hunk 1]
```diff
@@ -2094,8 +2094,46 @@ def encode(self, encoding, errors="strict"):
...
+    each string in the Series/Index from %(side)s. Equivalent to
+    :meth:`str.%(method)s`.
+
+    Parameters
+    ----------
+    to_strip : str or unicode
+        String or unicode to strip in the given string.
+
+    Examples
+    --------
```
Comment (datapythonista) [inline pandas/core/strings.py]: `Examples` section goes after `Returns`

File: pandas/core/strings.py [hunk 2]
```diff
@@ -2094,8 +2094,46 @@ def encode(self, encoding, errors="strict"):
...
+    >>> # strip method
```
Comment (datapythonista) [inline pandas/core/strings.py]: You can write sentences explaining what you're doing without making then Python comments. Check the docstring guide or other docstrings already improved.

File: pandas/core/strings.py [hunk 3]
```diff
@@ -2094,8 +2094,46 @@ def encode(self, encoding, errors="strict"):
...
+    >>> s = pd.Series(['   This is a Test 1  '])
```
Comment (datapythonista) [inline pandas/core/strings.py]: I'd create a single Series with all the values you want to use in the examples, and then apply all operations to it.

File: pandas/core/strings.py [hunk 4]
```diff
@@ -2094,8 +2094,46 @@ def encode(self, encoding, errors="strict"):
         return self._wrap_result(result)
 
     _shared_docs['str_strip'] = ("""
-    Strip whitespace (including newlines) from each string in the
-    Series/Index from %(side)s. Equivalent to :meth:`str.%(method)s`.
+    Strip whitespaces from string in Series.
```
Comment (datapythonista) [inline pandas/core/strings.py]: Not necessarily whitespaces.

**Comment by datapythonista** [comment]
Seems like #20863 is also fixing the same docstring, and IMO looks better. I think we can close this one.

**Comment by mroeschke** [comment]
Thanks @techedlaksh. Unfortunately it appears that there was some overlap with #20863; sorry about that. Feel free to work on other docstrings, always appreciated!

**Pull request closed without merging**